src/signals/incident-management/containers/FilterTagList/index.js

```diff
@@ -64,10 +61,6 @@ export const FilterTagListComponent = props => {
     category_slug: sub,
   };
 
-  if (!tags) {
```

Comment:
Why remove this check? The component shouldn't render anything when there are no tags to render.

Reply:
tags is in the defaultProps.

Reply:
It can still be bypassed. `defaultProps` Is merely a safeguard during development.